<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>831</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Processos</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>821</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>PID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditPID"/>
       </item>
       <item>
        <widget class="QPushButton" name="botaoSuspender">
         <property name="text">
          <string>Suspender</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botaoContinuar">
         <property name="text">
          <string>Continuar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botaoMatar">
         <property name="text">
          <string>Matar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botaoAlterarCPU">
         <property name="text">
          <string>Alterar CPU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botaoMudarPrioridade">
         <property name="text">
          <string>Mudar Prioridade</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tabela">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>811</width>
        <height>411</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>PID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>USER</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>#CPU</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>%CPU</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>%MEM</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>STATUS</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PRIO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CMD</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>511</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Filtro</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditFiltro">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Prioridade</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxPrioridade">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>50</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>CPU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxCPU">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gráficos</string>
     </attribute>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>231</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Gráfico de processos</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>661</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>CPU 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="asd2">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>CPU 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="asd">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GERENCIADOR DE PROCESSOS</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>botaoSuspender</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>suspender()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botaoContinuar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>continuar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botaoMatar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>matar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botaoMudarPrioridade</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mudarPrioridade()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>821</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botaoAlterarCPU</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>alterarCPU()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>662</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEditFiltro</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>filtro()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabela</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>pegarPIDTabela()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabela</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>textEditPID</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabela</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>copiaTexto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>578</x>
     <y>595</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>asd</sender>
   <signal>valueChanged(int)</signal>
   <receiver>asd</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>asd2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>asd2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>suspender()</slot>
  <slot>matar()</slot>
  <slot>continuar()</slot>
  <slot>mudarPrioridade()</slot>
  <slot>alterarCPU()</slot>
  <slot>pegarPIDTabela()</slot>
  <slot>filtro()</slot>
  <slot>slot2()</slot>
  <slot>copiaTexto()</slot>
 </slots>
</ui>
